fiftyone/utils/data/importers.py

```
@@ -1990,7 +2116,21 @@ def _import_samples(self, dataset, dataset_dict, tags=None, progress=None):
         dataset_id = dataset._doc.id
 
         def _parse_sample(sd):
-            if not os.path.isabs(sd["filepath"]):
+            media_reference = sd.get("_media_reference")
+            if media_reference is not None:
+                if not media_reference_revision:
+                    raise MediaReferenceError(
+                        "Native media-reference imports must declare dataset "
+                        "revision %s" % MEDIA_REFERENCE_DATASET_REVISION
```

Ruff (0.16.2)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)